gulpfile.js

```diff
@@ -151,7 +131,7 @@ gulp.task('subjects', function(done) {
                 dest: '',
                 render: {
                     scss: {
-                        dest: '_subjects-icons-embed.scss'
+                        dest: '_subject-icons-embed.scss'
```

Comment:
I renamed some files for consistency, this is internal, no changes to css clasess were made